### PR TITLE
Implementing IConvertible

### PR DESCRIPTION
**Pull request by tmilnthorp**

### CODE REVIEW

File: UnitsNet.WindowsRuntimeComponent/GeneratedCode/Quantities/Acceleration.WindowsRuntimeComponent.g.cs [hunk 1]
```diff
@@ -784,5 +784,107 @@ private static IFormatProvider GetFormatProviderFromCultureName([CanBeNull] stri
         {
             return cultureName != null ? new CultureInfo(cultureName) : (IFormatProvider)null;
         }
+
+        #region IConvertible Methods
+
+#if !WINDOWS_UWP
```
Comment (angularsen) [inline UnitsNet.WindowsRuntimeComponent/GeneratedCode/Quantities/Acceleration.WindowsRuntimeComponent.g.cs]: Can't we skip/remove this entire section in WRC files? I think we have a `$wrc` variable in powershell you can use.

**Comment by angularsen** [comment]
Ok, this is kinda neat! A bit unsure about converting to numbers like this, but hey, I guess it's the caller's funeral if they convert to number type and expected a different value :-) Is there any good way to communicate what to expect here? I don't see where to put that xmldoc though. Remarks on the class maybe?

File: UnitsNet.WindowsRuntimeComponent/GeneratedCode/Quantities/Acceleration.WindowsRuntimeComponent.g.cs (hunk 1 above)
Comment (tmilnthorp) [reply UnitsNet.WindowsRuntimeComponent/GeneratedCode/Quantities/Acceleration.WindowsRuntimeComponent.g.cs]: Good call. Done!

**Comment by angularsen** [comment]
Ok nice, there's a merge conflict here, but other than that you can just merge it.

**Comment by tmilnthorp** [comment]
Sounds good.

**Comment by angularsen** [comment]
[Release UnitsNet/4.8.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.8.0)

**Pull request merged**